.changes/unreleased/tinymce-TINY-10389-2024-01-23.yaml

```diff
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Removed
```

Comment:
I think this should be `Fixed` since `Removed` is more about removing APIs etc more radical changes this removal seems more safe since no one should be dependent on us firing two events here.